docs/SPEC.md

```diff
@@ -1132,6 +1132,12 @@ These are builtin functions that all take a single `time` argument and return an
 * `month` int
     Month returns the month of the year for the provided time in the range `[1-12]`.
 
+#### truncate
+
+`date.truncate` takes in a time t and a Duration unit and returns the given time 
+truncated to the given unit.
+
+Example: `truncate(t: "2019-06-03T13:59:01.000000000Z", unit: 1s)` returns time `2019-06-03T13:59:01.000000000Z`
```

Comment:
Could you give a more insightful example? For  example with minute or hour?

If you add examples as in your unit tests, that would be great!